content/tutorials/manage-the-rippled-server/installation/install-rippled-on-centos-rhel-with-yum.md

```diff
@@ -1,109 +1,17 @@
 ---
 html: install-rippled-on-centos-rhel-with-yum.html
 parent: install-rippled.html
-blurb: Install a precompiled rippled binary on CentOS or Red Hat Enterprise Linux.
+blurb: Deprecated: CentOS and Red Hat Enterprise Linux are no longer supported.
 labels:
```

Comment:
You can add "status: removed" to the frontmatter to give the page a red trash can icon in the table of contents (like with the Demurrage page) so it's more obvious that it's deprecated/removed.

```suggestion
status: removed
labels:
```